omod/pom.xml

```diff
@@ -130,60 +98,18 @@
 
 		<dependency>
 			<groupId>org.openmrs.module</groupId>
-			<artifactId>emrapi-api</artifactId>
-			<version>${emrapiVersion}</version>
-			<scope>provided</scope>
-		</dependency>
-
-		<dependency>
-			<groupId>org.openmrs.module</groupId>
-			<artifactId>emrapi-api-1.10</artifactId>
-			<version>${emrapiVersion}</version>
-			<scope>provided</scope>
-		</dependency>
-
-		<dependency>
-			<groupId>org.openmrs.module</groupId>
-			<artifactId>providermanagement-api</artifactId>
-			<version>${providermanagementVersion}</version>
-			<scope>test</scope>
-		</dependency>
-
-		<dependency>
-			<groupId>org.openmrs.module</groupId>
-			<artifactId>reporting-api</artifactId>
-			<version>${reportingVersion}</version>
-			<scope>test</scope>
-		</dependency>
-
-		<dependency>
-			<groupId>org.openmrs.module</groupId>
-			<artifactId>calculation-api</artifactId>
-			<version>${calculationVersion}</version>
-			<scope>test</scope>
-		</dependency>
-
-		<dependency>
-			<groupId>org.openmrs.module</groupId>
-			<artifactId>serialization.xstream-api</artifactId>
-			<version>${serialization.xstreamVersion}</version>
-			<scope>test</scope>
-		</dependency>
-
-		<dependency>
-			<groupId>org.openmrs.module</groupId>
-			<artifactId>serialization.xstream-api-1.10</artifactId>
-			<version>${serialization.xstreamVersion}</version>
-			<scope>test</scope>
-		</dependency>
-
-		<dependency>
-			<groupId>org.openmrs.module</groupId>
-			<artifactId>webservices.rest-omod</artifactId>
+			<artifactId>webservices.rest-omod-2.4</artifactId>
```

Comment:
Because we are supporting from version 2.4 of core, do we need a rest-omod-2.4 sub module?

Reply:
Yes @dkayiwa. For tests specifically.

Reply:
I do not seem to understand your response. My impression was that whatever you have in `omod-2.4` should simply be in `omod` because our base support is platform 2.4, according to the pull request title.

Reply:
Ohh, sorry I wasn't clear @dkayiwa.  You suggest pulling in `omod-2.4` transitively through `omod` in tests, but this breaks tests at compile time hence the need to explicitly declaring it as test-scoped dependency.

Reply:
I still do not get it. My understanding is that we create these submodules for supporting multiple platform versions. For instance, we create `webservices.rest-omod-2.4` for changes that require platform 2.4 and in which case this submodule would not be necessary if the base platform version that we support is 2.4

Reply:
We append the openmrs platform version or the omod name when supporting multiple platform versions. So `webservices.rest-omod-2.4` means changes which require platform 2.4 and hence cannot go into `webservices.rest-omod`. If as per the pull request title, you support from platform 2.4 and above, then i would not expect any sub module with 2.4 in its name.